freeipa.spec.in

```diff
@@ -57,8 +57,12 @@ Source0:        freeipa-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
 BuildRequires:  openldap-devel
+%if 0%{?fedora} > 25
+BuildRequires: krb5-devel >= 1.15-5
```

Comment:
@frozencemetery introduced a provides that expicitly marks DAL versions, can we use that provide instead of a version number ?